lib/config.js

```diff
@@ -16,8 +16,9 @@ exports.get = () => {
   try {
     customConfig = JSON.parse(fs.readFileSync(CUSTOM));
   } catch (ex) {
-    /*eslint-disable */
-    /*eslint-enable */
+    if(ex instanceof SyntaxError) {
```

Comment:
This kind of thing should be covered by eslint...

```suggestion
    if (ex instanceof SyntaxError) {
```

Reply:
The only thing the lint complains about are warnings about todos.

Reply:
Yeah, that's what I mean. We should update the earlier configuration to enforce uniform if statement usage.